orders/urls.py

```diff
@@ -0,0 +1,9 @@
+
+from django.urls    import path
+from orders.views   import OrderCheckView
+
+
+
+urlpatterns = [
+            path('/order', OrderCheckView.as_view())
+]
```

Comment:
```suggestion
urlpatterns = [
    path('', OrderCheckView.as_view())
]
```
이미 order가 있기 때문에 다음과 같이 수정해 주세요.

Reply:
수정하겠습니다~